backend/src/v2/objectstore/object_store.go

```diff
@@ -66,10 +72,10 @@ func OpenBucket(ctx context.Context, k8sClient kubernetes.Interface, namespace s
 		return blob.PrefixedBucket(openedBucket, config.Prefix), nil
 
 	}
-	return blob.OpenBucket(ctx, config.bucketURL())
+	return blob.OpenBucket(ctx, config.BucketURL())
 }
 
-func (b *Config) bucketURL() string {
+func (b *Config) BucketURL() string {
```

Comment:
This method is used only in the above function. Why are you making it public?

Reply:
good catch, this was a leftover from a different approach